src/brsTypes/components/RoSGNode.ts

```diff
@@ -25,6 +25,7 @@ import { ComponentFactory, BrsComponentName } from "./ComponentFactory";
 import { Environment } from "../../interpreter/Environment";
 import { roInvalid } from "./RoInvalid";
 import type * as MockNodeModule from "../../extensions/MockNode";
+import { BrsError } from "../../Error";
```

Comment:
Is this import used?

Reply:
Nope!  Removed — it probably snuck in while I was working on #522